src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -736,65 +668,126 @@ public T GetPayloadValue<T>(string key)
         }
 
         /// <summary>
-        /// Gets the 'value' of the 'jti' claim from the payload.
+        /// Tries to get the claim from the JWT payload.
         /// </summary>
         /// <remarks>
-        /// Provides a unique identifier for the JWT.
-        /// see: https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.7
-        /// <para>
-        /// If the 'jti' claim is not found, an empty string is returned.
-        /// </para>
+        /// The 'value' a type T if possible.
         /// </remarks>
-        public override string Id
+        /// <returns>true if successful, false otherwise.</returns>
+        public bool TryGetValue<T>(string key, out T value)
         {
-            get
+            if (string.IsNullOrEmpty(key))
             {
-                if (_id == null)
-                    _id = Payload.GetStringValue(JwtRegisteredClaimNames.Jti);
+                value = default;
+                return false;
```

Comment:
do we have tests covering these paths (for all of them), where the key is empty.